### PR TITLE
added unsend

### PR DESCRIPTION
**Pull request by LarisaBothaza**

### CODE REVIEW

File: src/main/java/socialnetwork/controller/AccountUserController.java [hunk 1]
```diff
@@ -85,6 +89,10 @@ public void deleteFriendship() {
 
             tableViewAccountUser.getSelectionModel().clearSelection();
         }
+        else{
+            Alert alert = new Alert(Alert.AlertType.ERROR, "Nothing selected");
+            alert.show();
```
Comment (BreabanSergiu) [inline src/main/java/socialnetwork/controller/AccountUserController.java:102]: insert java doc

Comment (LarisaBothaza) [reply src/main/java/socialnetwork/controller/AccountUserController.java:102]: bine

Comment (LarisaBothaza) [reply src/main/java/socialnetwork/controller/AccountUserController.java:102]: Am modificat :)

**Review by BreabanSergiu — APPROVED** [review]
cam Putin

**Pull request merged**